package.json

```diff
@@ -29,18 +29,8 @@
     "license": "MIT",
     "icon": "icon.png",
     "activationEvents": [
-        "onCommand:vscode-pets.start",
```

Comment:
Activation events are generated automatically from VsCode so do not need to be specified in the package.json.